cookbook/case_studies/ml_training/pytorch/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _pytorch-training:
+
+Train a Model Using PyTorch
+---------------------------
+
+**PyTorch** is a machine learning framework that accelerates the path from research prototyping to production deployment.
+You can build *Tensors* and *Dynamic neural networks* in Python with strong GPU acceleration using PyTorch. 
```

Comment:
Strong?

Reply:
I've copied the content from PyTorch's GitHub repo.